onlineapi/src/test/java/com/codingblocks/onlineapi/api/OnlineJsonApiAuthenticatedTest.kt

```diff
@@ -50,9 +52,20 @@ class OnlineJsonApiAuthenticatedTest {
 
     @Test
     fun `GET wishlist`() {
-        val wishlist = runBlocking { CBOnlineLib.onlineV2JsonApi.getWishlist().body()?.get() }
-        assertNotNull(wishlist)
-        assertTrue(wishlist!!.isNotEmpty())
+        val wishlistAdd = runBlocking { CBOnlineLib.onlineV2JsonApi.addWishlist(Wishlist(Course("103"))).body() }
+        assertNotNull(wishlistAdd)
+        assertTrue(wishlistAdd!=null)
```

Comment:
can you check something other than null because this doesn't verify that item was added into a wishlist and same for other tests as well

Reply:
Responses of above API's are as follows:

Add Wishlist API: This returns the course if and the wishlist id, since API return new wishlist id for every course we cannot verify with it so I will check if the course id is same or not

Get Wishlist API: Since it returns the list of the wishlist, so we can check whether this list is empty or not.

Check If Wishlisted API: This returns null if the course is not wishlisted by the user, checking if the response id null or not will do the work.

RemoveWishlist: This always returns null when user deletes the course from wishlist, if it doesn't returns null then it may be a problem.